actionpack/lib/action_controller/metal/exceptions.rb

```diff
@@ -3,12 +3,19 @@ class ActionControllerError < StandardError #:nodoc:
   end
 
   class BadRequest < ActionControllerError #:nodoc:
-    attr_reader :original_exception
-
     def initialize(msg = nil, e = nil)
+      if e
+        ActiveSupport::Deprecation.warn("Passing #original_exception is deprecated and has no effect. " \
+                                        "Exceptions will automatically capture the original exception.", caller)
+      end
```

Comment:
removed in https://github.com/rails/rails/commit/bc6c5df4699d3f6b4a61dd12328f9e0f1bd6cf46 ( noting because I was just starting to use original_exception in my logging when I noticed it swallowing the original exception.... )  `¯\_(ツ)_/¯`